managed/services/supervisord/logs.go

```diff
@@ -146,13 +148,23 @@ func (l *Logs) files(ctx context.Context) []fileContent {
 		})
 	}
 
+	b, m, err := readFile("/srv/alertmanager/alertmanager.base.yml")
+	if err != nil {
+		logger.Get(ctx).WithField("component", "logs").Error(err)
+	}
+	b, err = maskSensitiveValues(b)
```

Comment:
I think we should wrap it to `else` section, otherwise we will include error from this function in `fileContent` instead of original one that we got from `readFile`